flyteadmin_config.yaml

```diff
@@ -52,6 +52,7 @@ scheduler:
     region: "my-region"
     scheduleRole: "arn:aws:iam::abc123:role/my-iam-role"
     targetName: "arn:aws:sqs:my-region:abc123:my-queue"
+    scheduleNamePrefix: "flyte"
```

Comment:
can we make this optional? and the default can be `flyte`

Reply:
It is optional, if it's empty we just don't join "<prefix>_" with the identifier hash

Reply:
but i think having "flyte_" is better than just an identifier hash. In some organizations, this will help them figure out how the schedules were created

Reply:
the schedule description includes a descriptive identifier that pinpoints it's a workflow execution e.g. `Schedule for Project:flytekit Domain:production Name:workflows.app.MyWorkflow`